src/setup/rgw.py

```diff
@@ -473,7 +477,10 @@ def _update_hax_endpoint(conf: MappedConf, data_pod_hostname: str):
     def _update_hax_endpoint_and_create_admin(conf: MappedConf):
         """Update motr_ha(hax) endpoint values to rgw config file."""
         current_data_node = socket.gethostname().replace('server', 'data')
-        Rgw._update_hax_endpoint(conf, current_data_node)
+        try:
+            Rgw._update_hax_endpoint(conf, current_data_node)
+        except Exception:
+            pass
```

Comment:
if exception occurs, logic should jump to loop trying out all hax endpoints

Reply:
done